client/components/additionalFeatures.jsx

```diff
@@ -43,14 +58,14 @@ const AdditionalFeatures = (props) => {
 
   return (
     <StyledAdditionalFeatures>
-      <h2 style={{textAlign: 'center'}}>{header}</h2>
+      <H2 style={{textAlign: 'center'}}>{header}</H2>
```

Comment:
Did this style not work in your style component code? Is there a reason for setting this CSS rule inline?

Reply:
In the few instances where I did this, I wanted line-specific styling. But this is not one of those cases. Thanks for pointing this out. I have made a new commit to fix this! 👍